### DISCUSSION TITLE
Is their a way to disable specific plugins or the other way around only enable specific

### DISCUSSION THREAD

**Discussion by ervet** [danny-avila/LibreChat#2461] [General]
Hello,

thx all for LibreChat - is their a way to disable specific plugins from the Store ? For example if I just want Google Search and the Calculator.

I have seen that nearly everything is located in /api/app/clients/tools. 

Has anybody done this before ? Or will it even work

Greetings
Erik

**Comment by fuegovic** [comment]
Yes, for now you can manually remove them, see:
https://github.com/danny-avila/LibreChat/discussions/2132#discussioncomment-8833958

A proper solution is planned though

**Reply by fbaeumer** [reply]
This is definitely required.

**Reply by danny-avila** [reply]
@fbaeumer It's already implemented:

https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/config#filteredtools
